config/routes.rb

```diff
@@ -1,36 +1,40 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
```

Comment:
LMAO I love how we took the time to change this to single quotes, only because after the lesson today it will not matter